pkg/agent/controller/globalingressip.go

```diff
@@ -0,0 +1,113 @@
+/*
+SPDX-License-Identifier: Apache-2.0
+
+Copyright Contributors to the Submariner project.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package controller
+
+import (
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/klog"
+)
+
+const (
+	ClusterIPService           = "ClusterIPService"
+	headlessServicePod         = "HeadlessServicePod"
+	IPAllocated                = "Allocated"
+	defaultReasonIPUnavailable = "ServiceGlobalIPUnavailable"
+	defaultMsgIPUnavailable    = "Service doesn't have a global IP yet"
```

Comment:
Some of these constants are global/exported and some not. Is this intentional?

Reply:
Global ones are what I use in UT.